src/components/MessagesList/MessagesGroup/MessagesSystemGroup.spec.js

```diff
@@ -0,0 +1,479 @@
+							displayName: 'actor one',
+							type: ATTENDEE.ACTOR_TYPE.USERS,
+						},
+						user1: {
+							id: 'actor-2',
+							displayName: 'actor two',
+							type: ATTENDEE.ACTOR_TYPE.USERS,
+						},
+					},
+				},
+				previousMessageId: 90,
+				nextMessageId: 200,
+			})
+
+		})
+
+	})
+
+	describe('renders grouped system message of user actions', () => {
+		let MESSAGES
+		/**
+		 * @param {object} wrapper - Wrapper of the component
+		 * @param {string} expectedMessage - Expected message
+		 *
+		 */
+		function testGroupedSystemMessages(wrapper, expectedMessage) {
+			const messagesEl = wrapper.findAllComponents({ name: 'Message' })
+			// combined message doesn't include the last message (it has a different actor)
+			expect(messagesEl.at(0).props().message.message).toBe(expectedMessage)
+			expect(messagesEl.at(0).props().message.messageParameters).toStrictEqual({
+				actor: {
+					id: 'actor-1',
+					displayName: 'actor one',
+					type: ATTENDEE.ACTOR_TYPE.USERS,
+				},
+				user0: {
+					id: 'actor-4',
+					displayName: 'actor four',
+					type: ATTENDEE.ACTOR_TYPE.USERS,
+				},
+				user1: {
+					id: 'actor-5',
+					displayName: 'actor five',
+					type: ATTENDEE.ACTOR_TYPE.USERS,
+				},
+			})
+
+			expect(messagesEl.at(3).props().message.message).toBe(MESSAGES[2].message)
+			expect(messagesEl.at(3).props().message.messageParameters).toStrictEqual({
+				actor: {
+					id: 'actor-3',
+					displayName: 'actor three',
+					type: ATTENDEE.ACTOR_TYPE.USERS,
+				},
+				user: {
+					id: 'actor-6',
+					displayName: 'actor six',
+					type: ATTENDEE.ACTOR_TYPE.USERS,
+				},
+			})
+		}
+
+		beforeEach(() => {
+			MESSAGES = [{
+				id: 100,
+				token: TOKEN,
+				actorId: 'actor-1',
+				actorDisplayName: 'actor one',
+				actorType: ATTENDEE.ACTOR_TYPE.USERS,
+				message: 'actor-1 removed {user}',
+				messageType: 'system',
+				messageParameters: {
+					actor: {
+						id: 'actor-1',
+						displayName: 'actor one',
+						type: ATTENDEE.ACTOR_TYPE.USERS,
+					},
+					user: {
+						id: 'actor-4',
+						displayName: 'actor four',
+						type: ATTENDEE.ACTOR_TYPE.USERS,
+					}
+				},
+				systemMessage: 'user_removed',
+				timestamp: 100,
+				isReplyable: false,
+			}, {
+				id: 120,
+				token: TOKEN,
+				actorId: 'actor-1',
+				actorDisplayName: 'actor one',
+				actorType: ATTENDEE.ACTOR_TYPE.USERS,
+				message: 'actor-1 removed {user}',
+				messageType: 'system',
+				messageParameters: {
+					actor: {
+						id: 'actor-1',
+						displayName: 'actor one',
+						type: ATTENDEE.ACTOR_TYPE.USERS,
+					},
+					user: {
+						id: 'actor-5',
+						displayName: 'actor five',
+						type: ATTENDEE.ACTOR_TYPE.USERS,
+					}
+				},
+				systemMessage: 'user_removed',
+				timestamp: 200,
+				isReplyable: false,
+			}, {
+				id: 130,
+				token: TOKEN,
+				actorId: 'actor-3',
+				actorDisplayName: 'actor three',
+				actorType: ATTENDEE.ACTOR_TYPE.USERS,
+				message: 'Actor left the scene',
```

Comment:
Then also replace a text here. I know it doesn't affect tests, but can confuse someone in the future